chapter_convolutional-modern/alexnet_vn.md

```diff
@@ -139,21 +141,25 @@ a core group of researchers had dedicated themselves to this idea, attempting to
 The ultimate breakthrough in 2012 can be attributed to two key factors.
 -->
 
-*dịch đoạn phía trên*
+Trong khoảng thời gian cho đến khi xuất hiện bước đột phá tối tân cho các mạng chập nhiều tầng vào năm 2012, một nhóm các nhà nghiên cứu cốt cán đã theo đuổi ý tưởng này, cố gắng học các đặc trưng biểu diễn theo nhiều tầng của dữ liệu hình ảnh trong nhiều năm. 
+Nhưng để có được bước đột phá tối tân vào năm 2012 thì được quy cho hai yếu tố chính sau.
+
 
 <!--
 ### Missing Ingredient - Data
 -->
 
-### *dịch tiêu đề phía trên*
+### Yếu tố bị thiếu - Dữ liệu
 
 <!--
 Deep models with many layers require large amounts of data in order to enter the regime where they significantly outperform traditional methods based on convex optimizations (e.g., linear and kernel methods).
 However, given the limited storage capacity of computers, the relative expense of sensors, and the comparatively tighter research budgets in the 1990s, most research relied on tiny datasets.
 Numerous papers addressed the UCI collection of datasets, many of which contained only hundreds or (a few) thousands of images captured in unnatural settings with low resolution.
 -->
 
-*dịch đoạn phía trên*
+Mô hình học sâu với nhiều tầng đòi hỏi phải có một lượng dữ liệu lớn để có thể đạt được khả năng thể hiện vượt trội so với các phương pháp truyền thống dựa trên phương pháp tối ưu lồi (ví dụ: phương pháp hồi quy và kernel). 
```

Comment:
```suggestion
Mô hình học sâu với nhiều tầng đòi hỏi phải có một lượng dữ liệu lớn để có thể vượt trội so với các phương pháp truyền thống dựa trên tối ưu lồi (ví dụ: phương pháp hồi quy và các hàm kernel). 
```

Reply:
@cuongvng vẫn giữ lại `đạt được khả năng thể hiện` thay vì `để có thể vượt trội ` được không anh. Bởi vì ở đây nó thể hiện cho việc vượt trội về độ chính xác.

Reply:
Mà chỗ này đúng là đang nói về performance mà đâu chỉ nhắc riêng tới học biễu diễn ko đâu @DacDinh147 ?
```suggestion
Mô hình học sâu với nhiều tầng đòi hỏi phải có một lượng dữ liệu lớn để có thể vượt trội hơn đáng kể so với các phương pháp truyền thống dựa trên phương pháp tối ưu lồi (ví dụ: phương tuyến tính và hạt nhân (_kernel_)). 
```

Reply:
how about this
```suggestion
Mô hình học sâu với nhiều tầng đòi hỏi phải có một lượng dữ liệu lớn để có hiệu quả vượt trội so với các phương pháp truyền thống dựa trên phương pháp tối ưu lồi (ví dụ: phương pháp tuyến tính và phương pháp nhân). 
```

Reply:
Cám ơn mọi người đã góp ý ạ.